tonic_validate/metrics/response_length_metric.py

```diff
@@ -9,16 +9,15 @@
 
 
 class ResponseLengthMetric(BinaryMetric):
-    """Checks that response is within a certain number of characters."""
-
     def __init__(
         self,
         name: str,
         min_length: Optional[int] = None,
         max_length: Optional[int] = None,
     ):
         """
-        Create a metric that checks if the length of the LLM response is within a given length.
+        Create a binary metric that checks if the length of the LLM response is within a given length.
+        Returns 1 (True) if the length is within the given range, 0 (False) otherwise.
```

Comment:
Returns 1 (True) if the length is within the range. Returns 0 (False) if the length falls outside of the range.